src/System.Configuration.ConfigurationManager/src/System/Configuration/BaseConfigurationRecord.cs

```diff
@@ -3218,12 +3218,16 @@ internal static string NormalizeConfigSource(string configSource, IConfigErrorIn
             if (trimmedConfigSource.Length != configSource.Length)
                 throw new ConfigurationErrorsException(SR.Config_source_invalid_format, errorInfo);
 
-            if (configSource.IndexOf('/') != -1) // string.Contains(char) is .NetCore2.1+ specific
-                throw new ConfigurationErrorsException(SR.Config_source_invalid_chars, errorInfo);
-
             if (string.IsNullOrEmpty(configSource) || Path.IsPathRooted(configSource))
                 throw new ConfigurationErrorsException(SR.Config_source_invalid_format, errorInfo);
 
```

Comment:
Why did you move the condition below `if (string.IsNullOrEmpty(configSource) || Path.IsPathRooted(configSource))`

Reply:
Matter of preference I suppose. I thought it will better to first check if `Path.IsPathRooted` and then `ConfigPathUtility.IsValid`. Do you want me to move it?